frontends/common/options.cpp

```diff
@@ -89,6 +89,12 @@ CompilerOptions::CompilerOptions() : Util::Options(defaultMessage) {
     registerOption("--testJson", nullptr,
                     [this](const char*) { debugJson = true; return true; },
                     "Dump and undump the IR");
+    registerOption("--pi-file", "file",
+                   [this](const char* arg) { piFile = arg; return true; },
+                   "Write a PI control plane API description to the specified file.");
+    registerOption("--pi-as-json", nullptr,
+                   [this](const char* arg) { piAsJson = true; return true; },
+                   "Write out the PI API description as human-readable JSON.");
```

Comment:
The JSON output is honestly pretty readable, in my opinion. And you're exactly right: the thought was that using JSON would give us more flexibility to integrate with other tools, testing, etc.